controllers/cloud.redhat.com/providers/database/appinterface.go

```diff
@@ -162,10 +176,13 @@ func GetDbConfig(ctx context.Context, pClient client.Client, namespace, appName,
 	// The creds given by app-interface have elevated privileges
 	matched.Config.AdminPassword = matched.Config.Password
 	matched.Config.AdminUsername = matched.Config.Username
-	matched.Config.RdsCa = &rdsCa
+	bundle, ok := rdsCaBundles[rdsCaBundleURL]
```

Comment:
Slightly less complex
```
	b := map[string]string{"a": "b"}
	a := ""
	a, _ = b["c"]
```

thoughts? - removes `if` reduces complexity

Reply:
I was trying to avoid assigning a nil pointer but actually I don't think I need any of this. Looks like the value of `b["c"]` is already `""`